app/views/collections/_sort_and_per_page.html.erb

```diff
@@ -12,7 +12,7 @@
    <div class="sort-toggle">
      <%# kind of hacky way to get this to work on catalog and folder controllers.  May be able to simple do {action: "index"} but I'm not sure -%>
      <% unless @response.response['numFound'] < 2 %>
-        <%= form_tag collections.collection_path(@collection.noid), method: :get, class: 'per_page form-inline' do %>
+        <%= form_tag collections.collection_path(@collection.id), method: :get, class: 'per_page form-inline' do %>
           <%= label_tag(:sort, "<span>Sort By:</span>".html_safe) %>
```

Comment:
I think just passing the collection here to match with other paths might be good